End-to-End-Tests.md

```diff
@@ -319,52 +448,89 @@ If you don’t see the `Artifacts` link, go to the summary of the failing workfl
 
 ![Artifacts section in summary of e2e run](https://user-images.githubusercontent.com/52176783/118647358-502c9600-b7f2-11eb-9e41-a6f5f962ddfb.png)
 
-People on macOS -- Quicktime doesn’t seem to like the videos we generate, so you might need to use VLC media player to view the videos.
+Note for macOS: Quicktime doesn’t seem to like the videos we generate, so you might need to use VLC media player to view the videos.
 
 ## Metrics
 
-We track passes, known flakes, and failures that aren't known to be flakes (called "failures") using a logging server. You can view these metrics at https://oppia-e2e-test-results-logger.herokuapp.com.
+We track passes, known flakes, and failures that aren't known to be flakes (called "failures") using a logging server. You can view these metrics at https://oppia-e2e-test-results-logger.herokuapp.com. The code for this server is in the (private) [`oppia/oppia-e2e-test-results-logger`](https://github.com/oppia/oppia-e2e-test-restuls-logger) repository.
```

Comment:
Is it useful to add the last sentence? If so should we explain why it's private?

I lean slightly towards omitting it since it's not actually that useful, but your call.

Reply:
removed